{{cookiecutter.project_name}}/monitoring/model_monitoring_setup_template.py

```diff
@@ -0,0 +1,278 @@
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ### Optional Fields
+
+# COMMAND ----------
+
+# Name of the schema in which to create output tables. If provided
+# then this can be in either {{schema}} or {{catalog}}.{{schema}} format. If not provided, the
+# schema returned by `SELECT current_database()` is used.
+output_schema_name = None
+
+# The name of the column that holds the label for this row. This field is optional if you do not have labels to provide with your data.
+# This column will be used to compute model quality metrics
+label_col = None
+
+# The name of the column that holds the example ID for this row. This might be a unique identifier attached to each inference 
+# or some other identifier you don’t want analyzed as a feature. The provided column will be ignored during analysis. 
+example_id_col = None
+
+# A table containing baseline data for comparison. The baseline table is expected to match the schema of the monitored table
+# If columns are missing on either side then monitoring will use best-effort heuristics to compute the output metrics.
+baseline_table_name = None
+
+# data_monitoring_dir = "/Users/first.last@company.com" The absolute path to user-configurable directory for storing data monitoring assets.
+# The path must be pointing to an existing directory. If provided, the assets will be created under:
+#         "{{data_monitoring_dir}}/{{table_name}}"
+# Otherwise, the assets will be stored under the default directory:
+#         "/Users/{{user_name}}/databricks_data_monitoring/{{table_name}}"
+# Note that this directory can exist anywhere, including "/Shared/" or other locations outside of the /Users/ 
+# directory; this can be useful when configuring a production monitor intended to be shared within an organization.
+data_monitoring_dir = None
+
+# List of column expressions to slice data with for targeted analysis. The data is grouped by each expression independently,
+# resulting in a separate slice for each predicate and its complements. For example `slicing_exprs=["col_1", "col_2 > 10"]` will generate the
+# following slices: two slices for `col_2 > 10` (True and False), and one slice per unique value in `col1`. For high-cardinality columns, only the
+# top 100 unique values by frequency will generate slices.
+slicing_exprs = None
+
+# A list of custom metrics to compute alongside existing aggregate, derived, and drift metrics.
+custom_metrics = []
+
+# List of Databricks entity names that are associated
+# with this table. Only following entities are supported:
+#    ["models:/registry_model_name"] links a model registry model to the monitored table.
+linked_entities = ["models:/" + model_name]
+
+# COMMAND ----------
+
+# MAGIC %md
+# MAGIC 
+# MAGIC ## 1. Create the monitor
+# MAGIC - You can check **API Walkthrogh - Monitors** section from the [user guide](https://docs.google.com/document/d/1d7BUxbxKDnQfhKZ8e0w99by9JLF4gRmAvDjVxmpE0ZU/edit#heading=h.70vskvghxjux) for more explainations.
```

Comment:
Typically we would not like to link to private preview materials in these notebooks, and especially not this one since it's an internal google doc. I would scrap all references like this from the notebook